packages/components/atoms/f-spinner/src/components/Spinner.vue

```diff
@@ -1,32 +1,49 @@
 <template>
     <div
-        :class="$style['c-spinner']"
-        data-test-id="spinner">
-        I am a VSpinner Component (GB)
+        data-test-id="spinner-component">
+        <span
+            v-if="shouldShowSpinner"
+            :class=" $style['c-spinner']" />
+
+        <span :class="{ ['is-hidden']: shouldShowSpinner }">
```

Comment:
Sorry for the delay on my comment – just wanted to check what sort of tags you're wrapping here?

The reason I ask is that `span` tends to be for text-type inline content, whereas `div`is for more block level elements, so if this is wrapping a big block on content, I'd switch this wrapper tag to be a `div`.